Cargo.toml

```diff
@@ -69,7 +69,7 @@ sha2 = "0.10.8"
 similar-asserts = "1.5.0"
 stream_assert = "0.1.1"
 thiserror = "1.0.38"
-tokio = { version = "1.30.0", default-features = false, features = ["sync"] }
+tokio = { version = "1.39.1", default-features = false, features = ["sync"] }
```

Comment:
This change is necessary, otherwise `tokio::sync::watch::Sender` does not implement `Default`.